build.yaml

```diff
@@ -102,7 +102,22 @@ release:
                   find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \; 
                   rm -rf build
                   PHP_TOKENS=($(echo ${PHP_VERSION} | tr "-" " "))
-                  sudo ${PACKAGE_UTILITY_REMOVE} ${PHP_TOKENS[0]}-common
+                  # Installing packaging and testing connection
+                  sudo ${PACKAGE_INSTALL} packages/${PHP_TOKENS[0]}*.${PACKAGE_TYPE}
+                  php <<EOF
+                  <?php
+                  try{
+                    Cassandra::cluster()->build()->connect();
+                  } catch (Cassandra\\Exception\\RuntimeException \$e) {
+                    echo 'Caught exception: ',  \$e->getMessage(), "\n";
+                  }     
+            EOF
```

Comment:
nit: indent

Reply:
EOF doesnt work if the indent is deeper. Weird!